content/en/docs/Writing policies/verify-images.md

```diff
@@ -4,4 +4,145 @@ description: Check image signatures and add digests
 weight: 3
 ---
 
-TBD...
\ No newline at end of file
+{{% alert title="Note" color="warning" %}}
+Image verification is an **preview** feature. It is not ready for production usage and will likely change. 
+{{% /alert %}}
+
+[Sigstore](https://sigstore.dev/) is a [Linux Foundation project](https://linuxfoundation.org/) focused on software signing and transparency log technologies to improve software supply chain security. [Cosign](https://github.com/sigstore/cosign) is a sub-project that provides image signing, verification, and storage in an OCI registry.
```

Comment:
Is it safe to assume that all "modern" image registries are at least compliant enough with the OCI specification as to support Cosign? I realize it's kind of outside the scope for Kyverno, but it may help to inform users who find this feature potentially attractive that there *may* be some compatibility checks required.

Reply:
There is a link in the Cosign docs page: https://github.com/sigstore/cosign#registry-support.

Do you thing we should add a note and reference?

Reply:
Could just be as simple as linking to that URL from the "OCI registry" last two words of the final sentence. Or a note would be fine.

Reply:
Ok, added:

> Refer to the [Cosign documentation](https://github.com/sigstore/cosign#quick-start) for details, including [OCI registry support](https://github.com/sigstore/cosign#registry-support).